docs/dev-setup.md

```diff
@@ -27,10 +27,11 @@ install the operator-sdk tools.
     cd $GOPATH/src/github.com/metal3-io
     git clone https://github.com/metal3-io/baremetal-operator.git
     cd baremetal-operator
-    kubectl apply -f deploy/service_account.yaml
-    kubectl apply -f deploy/role.yaml
+    kubectl apply -f deploy/service_account.yaml -n metal3
+    kubectl apply -f deploy/role.yaml -n metal3
     kubectl apply -f deploy/role_binding.yaml
     kubectl apply -f deploy/crds/metal3_v1alpha1_baremetalhost_crd.yaml
+    kubectl apply -f deploy/operator.yaml -n metal3
```

Comment:
I'm sorry that I missed this before, but these instructions are showing how to run the baremetal-operator locally instead of lanuching it in the cluster, so this line shold be removed.  Step 5 below replaces this one.

Reply:
This line has been removed :)